### PR TITLE
add alphanet and test chainspects

### LINKED ISSUE #188: [Tech]: Set up Moonbase Alpha for HRMP channel with Moonbeam
### Problem statement

We want to open an HRMP channel with the Moonbeam parachain on Kusama. The Moonbeam team requires that we start with a Moonbase Alpha to ensure smooth operations while opening channels between our parachains.

### Description

**Guide**: https://docs.moonbeam.network/builders/interoperability/xcm/xc-registration/xc-integration/#moonbase-alpha-xcm

### Requirements

_No response_

<span data-href="#aha_task_list_start" data-version="v2" data-aha="Do not remove this tag. Used for Aha! syncing."/>

- [x] Environment set up

- [ ] Sync with Moonbeam team to sync on proceeding further
<span data-href="#aha_task_list_end" data-version="v2" data-aha="Do not remove this tag. Used for Aha! syncing."/>

Aha! Link: https://soramitsucoltd.aha.io/features/SKP-2

### PR DESCRIPTION
**Pull request by N1ghtStorm**

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -528,6 +544,81 @@ pub fn bridge_dev_config() -> ChainSpec {
     )
 }
 
+/// Config for bridge private testnet
+pub fn bridge_test_config() -> ChainSpec {
+    // Give your base currency a unit name and decimal places
```
Comment (Alexey-N-Chernyshov) [inline node/src/chain_spec.rs]:
```suggestion
    // base currency unit name and decimal places
```

File: node/src/chain_spec.rs [hunk 2]
```diff
@@ -528,6 +544,81 @@ pub fn bridge_dev_config() -> ChainSpec {
...
+        None,
+        // Protocol ID
+        Some("sora-test"),
+        // Fork ID
+        None,
+        // Properties
+        Some(properties),
+        // Extensions
+        Extensions {
+            relay_chain: "rococo-local".into(), // You MUST set this to the correct network!
```
Comment (Alexey-N-Chernyshov) [inline node/src/chain_spec.rs:616]:
```suggestion
            relay_chain: "alpha".into(),
```
?

File: node/src/chain_spec.rs [hunk 3]
```diff
@@ -528,6 +544,81 @@ pub fn bridge_dev_config() -> ChainSpec {
...
+            para_id: 2011,
```
Comment (Alexey-N-Chernyshov) [inline node/src/chain_spec.rs:617]: please, check `para_id` is correct

**Review by Alexey-N-Chernyshov — APPROVED** [review]

File: node/src/chain_spec.rs (hunk 2 above)
Comment (N1ghtStorm) [reply node/src/chain_spec.rs:616]: > ?
thank you for noticing! this is test env chainspec, it is not for alphanet

**Pull request merged**